Adapt to linux-wrs kernel version, which has character '_' inside.

Signed-off-by: Dongxiao Xu <dongxiao.xu@intel.com>

Upstream-Status: Inappropriate [embedded specific]

Index: dpkg-1.16.1.2/lib/dpkg/parsehelp.c
===================================================================
--- dpkg-1.16.1.2.orig/lib/dpkg/parsehelp.c
+++ dpkg-1.16.1.2/lib/dpkg/parsehelp.c
@@ -246,11 +246,11 @@ parseversion(struct versionrevision *rve
   if (*ptr && !cisdigit(*ptr++))
     return dpkg_put_warn(err, _("version number does not start with digit"));
   for (; *ptr; ptr++) {
-    if (!cisdigit(*ptr) && !cisalpha(*ptr) && strchr(".-+~:", *ptr) == NULL)
+    if (!cisdigit(*ptr) && !cisalpha(*ptr) && strchr(".-+~:_", *ptr) == NULL)
       return dpkg_put_warn(err, _("invalid character in version number"));
   }
   for (ptr = rversion->revision; *ptr; ptr++) {
-    if (!cisdigit(*ptr) && !cisalpha(*ptr) && strchr(".+~", *ptr) == NULL)
+    if (!cisdigit(*ptr) && !cisalpha(*ptr) && strchr(".+~_", *ptr) == NULL)
       return dpkg_put_warn(err, _("invalid character in revision number"));
   }
 
